### DISCUSSION TITLE
put marker to mouse click location

### DISCUSSION THREAD

**Discussion by t0mka14** [GSI-CS-CO/chart-fx#381] [Q&A] [answered]
Hello,
I would like to use chart-fx library as a part of our research application for pacient self examination where their voice is recorded. Basically i need something like very simple audio cutter. A have managed to plot recorded audio amplitude by using XYChart, but what i would like to do now is to put two vertical markers into the graph, which the user can move and select part of the waveform between them. And then finally retrieve their X axis position to cut the audio. I have looked into the examples, but considering the markers I have found only the LabelledMarkerSample, which just shows how to draw vertical line in the graph. Do you guys see any easy way how to implement such a feature?
Thanks for the reply and also for the great library!
Tomas

**Comment by RalphSteinhagen** [comment, marked as answer]
@t0mka14 thanks for your interest in chart-fx and for being one of the first users of the new discussions feature! :+1: :smile: 

We have similar types of applications and thus implemented an `XRangeIndicator` plugin for this that you could use. The boundaries can be dragged and have properties you could bind or listen to. Also, the lines and area between the boundary markers etc. are stylable.

You may want to have a look at the [ChartIndicatorSample](https://github.com/GSI-CS-CO/chart-fx/blob/master/chartfx-samples/src/main/java/de/gsi/chart/samples/ChartIndicatorSample.java) and other samples/plugins.

**Reply by t0mka14** [reply]
@RalphSteinhagen thanks for the quick and kind reply :D i will for sure have a look at that example 👍 

Considering the amount of great features this library has, the discussion will for sure come in handy, as it is sometimes hard to discover them :)

**Reply by RalphSteinhagen** [reply]
@t0mka14 hope this helped. :+1:

Yes, we, unfortunately, lack tutorials a bit and rely primarily on the samples, reading the class documentation, or since the beginning of the year the new GH discussion feature.

Enjoy, and please feel free to share your experience or project with others (e.g. on GH).
